src/opera/test/pge/cslc_s1/test_cslc_s1_pge.py

```diff
@@ -77,6 +77,16 @@ def setUp(self) -> None:
             f"touch {join(input_dir, 'db.sqlite3')}"
         )
 
+        os.system(
+            f"touch {join(input_dir, 'test_qa.py')}"
+        )
+        os.chmod(f"{join(input_dir, 'test_qa.py')}", stat.S_IRWXU)   # Set to read, write, execute by owner
```

Comment:
So its a little clearer what the purpose of these dummy python scripts are, can we change the name to `test_qa_rwx.py` for the version that has read/write/execute privileges, and set the other to `test_qa_ro.py` for the read-only version?